src/modules/logger/logger.cpp

```diff
@@ -314,6 +317,10 @@ Logger *Logger::instantiate(int argc, char *argv[])
 			}
 			break;
 
+		case 'a':
```

Comment:
I don't like `a` because it often stands for `all`. What about `x`?

Reply:
`x` is fine for me too. I changed it.